tests/intensive/test_visualization.py

```
@@ -726,13 +726,13 @@ def test_add_samples_umap_no_embeddings_field_raises():
 
     _add_new_samples(dataset, 5, 64, rng, populate_field=False)
 
-    # UMAP runs computed without an embeddings_field drop their reducer at
-    # compute time, so incremental updates are unavailable
+    # UMAP runs computed without an embeddings_field cannot support
+    # incremental updates
     try:
         results.add_samples(dataset, embeddings=np.zeros((5, 64)))
         assert False, "expected ValueError"
```

Ruff (0.16.1)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)